frontend/src/app/main/ui/components/forms.scss

```diff
@@ -84,6 +84,9 @@
   width: var(--input-width, calc(100% - $s-1));
   min-width: var(--input-min-width);
   height: var(--input-height, $s-32);
+  input {
```

Comment:
Let's avoid styling raw tags, and add a class to that `<input>` instead (also, see if we can have it un-nested)